src/Symfony/Component/Translation/Translator.php

```diff
@@ -311,6 +313,8 @@ public function getMessages($locale = null)
      */
     protected function loadCatalogue($locale)
     {
+        trigger_error('The '.__METHOD__.' method is deprecated since version 2.8 and will be removed in 3.0. Rely on the TranslatorBagInterface instead', E_USER_DEPRECATED);
```

Comment:
this now triggers deprecations in Symfony itself, which is bad.

what you need to do is to trigger the deprecation in caller places in case the method is overwritten (have a look at the way we did for ``setDefaultOptions`` in form types, or for ``Kernel::init`` even though this one was simpler as the default method does nothing)